src/components/Table/TablePagination/TablePagination.tsx

```diff
@@ -60,6 +60,7 @@ export const TablePagination: React.FC<TablePaginationProps> = ({
                 className={clsx(cssClass32)}
                 icon={<ChevronFirstPage/>}
                 variant="ghost"
+                aria-label="go-to-first-page"
```

Comment:
`aria-label` is mainly for assistive technology, so you should write `Go to the first page`. But this string will not be translated so maybe it will be better to use `data-sel-role` to target element in our tests